LabDrivers/utils.py

```diff
@@ -254,6 +265,135 @@ def is_IP_port(device_port, return_vals=False):
     else:
         return True
 
+def list_properties():
+    """
+        This returns a dictionary of all drivers that have properties dictionary in form:
+        return {
+            'NAME': object
+        }
+    """
+    interface = [INTF_VISA, INTF_PROLOGIX, INTF_SERIAL]
+
+    #    instruments.append('TIME')
+    properties = {}
+    #    params['TIME']=[]
+
+    # list all the .py files in the drivers folder
+    my_path = inspect.getfile(list_drivers).rstrip('utils.py')
+    driver_files = os.listdir(my_path)
```

Comment:
As you reuse these two lines of code in the function below (and obviously you copied it from the function `list_drivers` ^^), you could put it in a function `get_drivers_list()`. Another quote from my 1st year c++ teacher : "If you copy several lines of code more than once, why not writing them into a function ?"

Reply:
I tend to agree with your 1st year c++ teacher :P especially in this case. I created the function inside LabDrivers/utils.py, and replaced it the list_* functions